erpnext/accounts/doctype/accounts_settings/accounts_settings.json

```diff
@@ -29,6 +29,7 @@
   "submit_journal_entries",
   "tax_settings_section",
   "determine_address_tax_category_from",
+  "manipulate_grand_total_for_inclusive_tax",
```

Comment:
One detail: “Manipulating” something is a red-flag wording, associated with illegal behaviour.
This however is a perfectly legal and in many cases even necessary rounding adjustment. 🤷🏻‍♂️

Reply:
It was kept that way to match with the actual function name:
https://github.com/frappe/erpnext/blob/80ab4eea8c0fb4284e6328aea8bb0c886e14ea5e/erpnext/controllers/taxes_and_totals.py#L519

We could name it to something like `Forcefully match Grand Total for inclusive tax` if required. Thoughts?

Reply:
I see, and I think that one should be renamed, too…
Is this so overly forceful? It’s even enabled on default, right?

I’d personally go with “adjust” and “adjustment”:
How about `adjust_grand_total_for_inclusive_tax`?

Reply:
Updated!

![image](https://github.com/frappe/erpnext/assets/52111700/c62f8c32-6d60-459e-90de-7d832a00015a)